app/models/submission/representation.rb

```diff
@@ -15,7 +15,10 @@ def self.digest_ast(ast)
   has_one :exercise, through: :solution
 
   has_one :exercise_representation,
-    ->(sr) { where("exercise_representations.exercise_id": sr.solution.exercise_id) },
+    lambda { |sr|
+      where("exercise_representations.exercise_id": sr.solution.exercise_id).
+        order('exercise_representations.id desc')
```

Comment:
This is the main actual functional change. Which is nice!

Reply:
By virtue of this being a `has_one`, only one record is returned? Nice